### ISSUE TITLE
PreciseManeuver does not save settings?

### ISSUE THREAD

**Issue by JupiterJaeden** [radistmorse/KSPPreciseManeuver#23]
Playing on latest KSP version 1.10.2939. The majority of the mod amazingly works just fine, except for a little bit of occasional weirdness with trying to set a node to the AN/DN. However, there is one major issue with the mod, which is that the settings don't save! So every time I restart KSP I have to change my hotkeys and settings for the mod. Is there some way this can be fixed? It's not game breaking, and I think I prefer this mod over Precise Node, but if this persists I may have to end up switching.

**Comment by JupiterJaeden** [comment]
**TL;DR I began writing this post as I was doing my testing/documenting. During the course of this, I was able to magically fix the bug myself. You can scroll down to the end of this post to see that. Thanks for responding on the forums though!**

Here are some more details: 
KSP: Windows 64-bit, version 1.10.2939
KSP was installed through Steam, but I am using the latest version of CKAN to manage my mods and launch it. To try and reproduce this bug in a clean environment, I used CKAN to clone my main KSP instance and then uninstalled all mods. I also deleted my saves that had been cloned. But I did not change anything else, so anything that might have carried over from my original instance could still be there. I then (manually) reinstalled Precise Maneuver version 2.4.4.

After launching KSP, I started a new Sandbox game with default settings and launched a simple craft into LKO. I went into map view and configured the settings using the stock toolbar as I normally would. I also created and saved a simple maneuver preset. Then, I quit to main menu and left the game. 

Here is the Player.log file generated from that: 
[Player.log](https://github.com/radistmorse/KSPPreciseManeuver/files/5645567/Player.log)
Recording (I apologize for the incredibly shitty video quality; I didn't realize how bad it was until after I had already recorded): https://drive.google.com/file/d/1cZ5gaLqKLEhiiARr-2C2xpPgHXCq37FP/view?usp=sharing

Upon reloading the game, I loaded the saved sandbox game and took control of the craft. I went into map view and checked the settings in the stock toolbar. Magically, it was now working! There are only 2 relevant things I can think of that I might be different from my real game: 
- There is a mod conflict causing this bug
- Maybe when I saved the maneuver preset, it fixed the bug; I didn't save any maneuver presets in my real game. 

If you want, here is the Player.log file generated from that: [Player.log](https://github.com/radistmorse/KSPPreciseManeuver/files/5645654/Player.log)
Recording: https://drive.google.com/file/d/1GghDBNj_dFaEhOGCux186HcMEDghtfrm/view?usp=sharing

So it seems I will need to test this in my real game. I switched to my main instance of KSP and I was about to send the mod list when I realized that one of my mods (Maneuver Node Splitter) was not showing up in the CKAN list. This was kind of weird, so I went ahead and deleted it manually and then tried to install it through CKAN. Turns out I was missing a dependency! So I installed that, and now here is the mod list for that game: 
![Screenshot (974)](https://user-images.githubusercontent.com/48777952/101218217-ae085f80-3650-11eb-87f6-c87248d335b3.png)

The first thing I wanted to test is to see if saving a maneuver preset made it magically work. I loaded my main game, picked a random craft to switch to, then configured the settings as described earlier. I then quit to main menu and left the game. 

Player.log file: 
[Player.log](https://github.com/radistmorse/KSPPreciseManeuver/files/5645719/Player.log)
Recording: https://drive.google.com/file/d/1t6GW4zEBlBen2EaB7NwJNt0yI3kJIVyH/view?usp=sharing

Upon reloading the game, and switching back to the same vessel, all of my settings were magically saved. I made another recording for this, but I see no reason to upload it now. Nevermind, KSP works in mysterious ways apparently. Either installing the missing mod dependency for the other mod fixed it, or saving a single random maneuver preset fixed it. Regardless, it seems to work now and I can keep using this great mod. If it becomes an issue again, I will contact you. Have a great day!
